types/index.d.ts

```diff
@@ -846,22 +846,26 @@ export type Operation =
 
```

Comment:
Is `signPayloadDecorated(data: Buffer): xdr.DecoratedSignature;` missing in the file?

Reply:
Yes it is - nice catch! Could've sworn I added it..